### DISCUSSION TITLE
BNB Beacon unable to claim as BSC

### DISCUSSION THREAD

**Discussion by marinobucci** [bnb-chain/bsc#2837] [General]
I have BNB Beacon in my TrustWallet on my phone. 
![IMG_8487](https://github.com/user-attachments/assets/e39365e7-eb8a-4641-b9fb-c68c1216469f)
![IMG_8486](https://github.com/user-attachments/assets/9da20def-6fe7-4bba-a0dd-9b9af383ac7b)

But the Trust Wallet web browser extension does not show any BNB Beacon balance at all.  
<img width="1440" alt="Screenshot 2024-12-30 at 9 30 02 PM" src="https://github.com/user-attachments/assets/3519b599-2c75-4371-9a51-6ccd737dbec3" />

Therefore, when I attempt to use the https://www.bnbchain.org/en/token-recovery tool there is no BNB Beacon coin to recover shown, despite there actually being a balance in my wallet. I have tried disconnecting and reconnecting numerous times with no answer.

I transferred BNB from my Binance to my Trust Wallet BEP2 Address four times, the transaction already confirmed and it showed on my trust wallet but the asset did not show, it still shows zero balance. I already reinstalled the apps but still, the asset does not show, I've reconnected my wallets to the browser and nothing seems to work. 

I saw on the community forum to submit a ticket to customer support at https://support.trustwallet.com/ regarding this, but only the chat bot is available and is unable to help.

I have been scammed already, losing all my assets in TrustWallet because I was not able to get ahold of a real support agent. The only thing that remains in my Wallet is the BNB Beacon that I am unable to convert or do anything with. I am just trying to actually connect with a single human individual from Trust Wallet. 

Please advise, I am desperate.

https://explorer.bnbchain.org/tx/1DC22F5133B7BA6FB20E3F9F48680F041A79199C92AC86A0050ED38BB7150BC9

**Comment by unclezoro** [comment]
![image](https://github.com/user-attachments/assets/61b2baa5-40d7-4f20-88e5-c33ba7e9bf14)

The token already sent to your wallet on BSC 65 days ago
